internal/store/digest.go

```diff
@@ -0,0 +1,54 @@
+package store
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/docker/distribution/reference"
+	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
+)
+
+// We store bundles both by image:tags and by plain digest.
+//
+// Within the bundle store, the file layout is
+// <registry>
+//  \_ <repo>
+//       \_ <tag>
```

Comment:
Can we also add the structure in the store of a digested reference ?
```
\_ <digested-reference>
```

Reply:
There's no such thing (until someone badly hack the reference API :P)
Digested have to apply to a Named reference, i.e `foo/bar:@digest`